modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.h.mustache

```diff
@@ -14,6 +14,7 @@
 #include <QObject>
 #include <QString>
 #include <QTimer>
+#include <QRandomGenerator>
```

Comment:
The issue here is that this is only available in version 5.15. On older Qt5 versions the build will fail.
CI failed to build this too.

Reply:
Changed this to use `qrand()` and `qsrand()` if were not in Qt6.